aws-infra/modifications/storage.py

```
@@ -0,0 +1,161 @@
+# ---------------------------------------------------------------------
+# CREATE STORAGE SOURCE
+# ---------------------------------------------------------------------
+def create_storage_source(project: Project, name: str, prefix: str = S3_TEST_CONFIG.prefix) -> S3StorageSource:
+
+    # Try creating tiny placeholder to verify prefix
+    placeholder_key = f"{prefix}/.placeholder"
+
+    try:
+        s3.write_file(S3_TEST_CONFIG, placeholder_key, b"")
+        logger.info(f"[S3] Verified prefix exists: {placeholder_key}")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)